From d3543f695435f71e1de8e6a4abe9e491be546a10 Mon Sep 17 00:00:00 2001
From: "Uwe L. Korn" <uwe.korn@quantco.com>
Date: Thu, 15 Feb 2024 15:53:40 +0100
Subject: [PATCH] Don't install pylint_plugins

---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index c37b1c2..31a2bc6 100644
--- a/setup.py
+++ b/setup.py
@@ -125,7 +125,7 @@ skinny_package_data = [
 setup(
     name="mlflow" if not _is_mlflow_skinny else "mlflow-skinny",
     version=version,
-    packages=find_packages(exclude=["tests", "tests.*"]),
+    packages=find_packages(exclude=["tests", "tests.*", "pylint_plugins"]),
     package_data=(
         {"mlflow": skinny_package_data}
         if _is_mlflow_skinny
-- 
2.43.0

