From 5e57b12ef04dfc8963ffb0f3f1f15632c6603e0f Mon Sep 17 00:00:00 2001
From: "Uwe L. Korn" <uwe.korn@quantco.com>
Date: Thu, 18 Aug 2022 18:24:27 +0200
Subject: [PATCH] Ignore empty proxies

---
 mlflow/models/docker_utils.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mlflow/models/docker_utils.py b/mlflow/models/docker_utils.py
index 5b36462..9702935 100644
--- a/mlflow/models/docker_utils.py
+++ b/mlflow/models/docker_utils.py
@@ -38,7 +38,7 @@ RUN python /tmp/get-pip.py
 RUN pip install virtualenv
 """
 
-if os.getenv("http_proxy") is not None and os.getenv("https_proxy") is not None:
+if os.getenv("http_proxy") and os.getenv("https_proxy"):
 
     # Expects proxies as either PROTOCOL://{USER}:{PASSWORD}@HOSTNAME:PORT
     # or PROTOCOL://HOSTNAME:PORT
-- 
2.37.2

