From 2f03460c14a97ba90f1d951562ab1032879313cb Mon Sep 17 00:00:00 2001
From: harupy <hkawamura0130@gmail.com>
Date: Wed, 17 Aug 2022 18:22:57 +0900
Subject: [PATCH] Ignore empty environment variables in _get_maven_proxy

Signed-off-by: harupy <hkawamura0130@gmail.com>
---
 mlflow/models/docker_utils.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mlflow/models/docker_utils.py b/mlflow/models/docker_utils.py
index 3e384b48950..26c5d8f0276 100644
--- a/mlflow/models/docker_utils.py
+++ b/mlflow/models/docker_utils.py
@@ -42,7 +42,7 @@
 def _get_maven_proxy():
     http_proxy = os.getenv("http_proxy")
     https_proxy = os.getenv("https_proxy")
-    if http_proxy is None or https_proxy is None:
+    if not http_proxy or not https_proxy:
         return ""
 
     # Expects proxies as either PROTOCOL://{USER}:{PASSWORD}@HOSTNAME:PORT
